### ISSUE TITLE
Popover content property accepts not just string and function

### ISSUE THREAD

**Issue by Herst** [twbs/bootstrap#16860]
The popover content option would apparently also accept any content that jQuery's .html() accepts and not just a string or function, see https://jsfiddle.net/Herst/bgc3puuf/

Is there any reason why the documentation does not reflect that?
http://getbootstrap.com/javascript/#popovers-options

**Comment by twbs-lmvtfy** [comment]
Hi @Herst!

You appear to have posted a live example (https://fiddle.jshell.net/Herst/bgc3puuf/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 15, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.4 ; saw what appears to be usage of Bootstrap 3.0.0
* line 19, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.4 ; saw what appears to be usage of Bootstrap 3.0.0

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by Herst** [comment]
Oops, here a revisited example: https://jsfiddle.net/Herst/bgc3puuf/

**Comment by mdo** [comment]
> Is there any reason why the documentation does not reflect that?

Nope—if you have a suggested edit, please feel free to propose in a PR. Otherwise, since this isn't a critical fix, I'm closing as a won't fix on our end.

**Comment by Herst** [comment]
Just to clarify: My question was also about whether there could be any unforeseen side-effects.

**Comment by Herst** [comment]
Incidentally fixed by c7d8e7a.
